### DISCUSSION TITLE
2022-10-26 Weekly Meeting 79 Agenda

### DISCUSSION THREAD

**Discussion by antpb** [omigroup/omigroup#307] [Weekly Meetings]
Weekly Meeting 79 is happening on Wednesday October 26, 2022 from 9:00 AM PST(16:00 UTC) - 10:00 AM PST (17:00 UTC)

You can add it to your calendar [here](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t).
 
We'll be meeting again in the omi-weekly-meeting channel of the [AngellXR/OMI Discord](https://discord.gg/NJtT9grz5E).
We regularly use the # omi-meeting-chat channel for text chat. 

## Who we are and why we're here:

The Open Metaverse Interoperability Group (OMI) is focused on bridging virtual worlds by designing and promoting protocols for identity, social graphs, inventory, and more. Our members include businesses and individuals working towards this common goal. 

If you’re an observer or here for the first time, this is a meeting where people who work on the OMI group initiatives discuss updates and contribution opportunities for current projects and goals. This is your chance to jump in and help!

We welcome you to introduce yourself in #omi-meeting-chat as we dive into the agenda items. There will be an open floor in the last 15 minutes of the meeting (agenda permitting) where we can discuss any items you feel are relevant to the current or near future OMI goals.

## Agenda
- Volunteer to take notes? Volunteer to record? 
- Welcome newcomers

### Working Group Updates:
- [MSF delegates](https://github.com/omigroup/MSF-Delegates) (setting up the repo)
- [UX Research](https://github.com/omigroup/ux-research) 
- [Scripting group](https://github.com/omigroup/omi-scripting-group)
  - OMI developer hour* 
- [glTF Extensions Group](https://github.com/omigroup/gltf-extensions)
  - Progress on OMI_ref and authoring extensions
- [Media Group](https://github.com/omigroup/media)
- [M3 avatar interop](https://github.com/M3-org/avatar-interop#calendar)
- [OMI Website update](https://github.com/omigroup/website)
- any others?

### Discussions: 
- a note on how we respect and communicate with others. 
- #308 
- #309 
- https://github.com/omigroup/omigroup/milestone/1
- Adding OMI to the street Jin shared last week. 
- https://github.com/omigroup/website/issues/59

### Future demo themes: 
- `OMI-audio-emitter` exploration and demo 
- `OMI-collider` exploration and demo 
- `Generative and AI tooling`
- Any other suggestions?

**Comment by pcsmgnt** [comment]
Not confirmed, but a rep from Tech Circus may be coming to speak about their upcoming event in November. A new issue has been created for this

**Reply by mrmetaverse** [reply]
#308

**Reply by mrmetaverse** [reply]
## Tech Circus "Enter the Metaverse" 

Tech Circus has an event coming up November 28th and 29th, called "enter the Metaverse". 

Discussing things like gaming, Interoperability, scalability, UX/UI, etc.

They are seeking speakers to attend the event in London. If there is interest in attending the event they would love to speak with us as well.

**Reply by pcsmgnt** [reply]
<img width="453" alt="image" src="https://user-images.githubusercontent.com/51180417/198144368-4f97503b-40dd-43af-9679-f7401d9333ca.png">

myles@techcircus.co.uk

**Comment by mrmetaverse** [comment]
# MSF - Delegates [repo](https://github.com/omigroup/MSF-Delegates)
A new Plenary meeting occurred this morning. 
- several new MSF opportunities emerged 
  - a new interoperable avatar proposal was drafted as well 

Meta: 
- Delegates have been meeting to reflect on how they can meet the needs of gaps they are having. 
- if you would like a recording please reach out in the channel to @humbletim

If there is interest in getting involved as a delegate, please reach out as well. They were recently able to add Sindhu to the MSF delegates.

**Reply by pcsmgnt** [reply]
New MSF proposals from the community include:
* Jobs Board
* Exploratory Group Matching-Making Assistance

I think both are great!

**Comment by mrmetaverse** [comment]
# UX Research 
- Newest group, still forming our repo, etc. 
- Next meeting is on November 1, 2022 at 16:00 UTC
- Do you have an idea for things we should research? Where should we start? https://github.com/omigroup/ux-research/issues/12
  - Next meeting we will review and discuss!

**Comment by mrmetaverse** [comment]
# Scripting 

Next meeting is tomorrow at 17:00 UTC.

# DevHour 

Next meeting is next week on Thursday.

**Comment by mrmetaverse** [comment]
# gLTF 



## Demo 

![image](https://user-images.githubusercontent.com/63426722/198081437-9d86005e-bdd7-4b6a-bd40-30e57989800d.png)

using thirdroom exporter 

![image](https://user-images.githubusercontent.com/63426722/198081478-4f6c64b9-973d-4277-94ba-78e406828ad5.png)

Gives a nice interface to show what the values will look like for audio. Green is what you will hear, and yellow what you won't

![image](https://user-images.githubusercontent.com/63426722/198081573-84e79799-1e3f-4ee8-b7b8-bfc612b5bb9a.png)

When it's all set up in a space, the third room exporter will package everything for you 

![image](https://user-images.githubusercontent.com/63426722/198081650-612aaa0b-8f60-4f0b-b94b-e8ec156c5f5c.png)

threeomi loader 

![image](https://user-images.githubusercontent.com/63426722/198081698-bbc7f82e-58f8-476d-883e-6bd470dbd0a9.png)

just add the audio extension, which will parse the glb file, and the audio will respond. 

You can 

![image](https://user-images.githubusercontent.com/63426722/198081802-e7904815-24e1-492a-9a46-632025efd715.png)

![image](https://user-images.githubusercontent.com/63426722/198081862-7078a43d-a557-4ec1-bfd9-9289668f1e8b.png)

![image](https://user-images.githubusercontent.com/63426722/198081959-c4ef0448-d59d-4d5d-b97b-acb973f893b0.png)

spatial audio 

![image](https://user-images.githubusercontent.com/63426722/198082039-891a9e79-250a-4c1b-9562-8929cecfc58e.png)

Other screens: 

![image](https://user-images.githubusercontent.com/63426722/198082731-cd92b783-21a6-4479-a3ae-662e72d4c95f.png)

![image](https://user-images.githubusercontent.com/63426722/198082768-8dd04866-193a-48af-a6fc-23dcf6a184d1.png)


You can export omi khr audio extensions. 

In react-three-fiber if you want to support OMI audio, just add the three-omi package. 

![image](https://user-images.githubusercontent.com/63426722/198082918-e9c8a93c-4333-4b32-94d3-f9c344fbb03c.png)

Colliders are where it gets different: 

![image](https://user-images.githubusercontent.com/63426722/198083014-a9b67ba4-04d4-4def-b37a-0f32d4ff2bb2.png)

[additional information](https://twitter.com/antpb/status/1583613326466068481)

**Reply by mrmetaverse** [reply]
lol I tried to keep up :sweat_smile:

**Reply by mrmetaverse** [reply]
>We could start demoing these extensions using our own Wordpress site too!

**Reply by mrmetaverse** [reply]
Networking will be enabled in the future so we can have a field trip into the site.

**Comment by mrmetaverse** [comment]
# Draft Events 101 with OMI policy 

Sindhu, Evo, and others have been working together to draft a draft proposal for #301 

Proposes items like having a speakers bureau and more.

![image](https://user-images.githubusercontent.com/63426722/198086250-3a2d6b75-5160-47ac-b6bb-df9c90b6679e.png)

**Reply by mrmetaverse** [reply]
Morpheus has volunteered to help out as well. They have experience in standards as well, and reference items if needed.

**Comment by mrmetaverse** [comment]
# Proposing a new Role: Someone to oversee and field events 

Sindhu and Evo discussed the need for a new potential champion that can oversee events.

**Reply by pcsmgnt** [reply]
Definitely needed and ties into the opportunities we are bringing back from the MSF Speakers' Bureau postings that are currently being shared in the MSF Opportunities channel on Discord. Alex Crabb manages these activities for MSF. I think the form @LightLodges ha suggested would be a great way to proactively match the demand/supply side of things.

**Comment by mrmetaverse** [comment]
# a note on how we respect and communicate with others

- Some have expressed concerns with trust and consent, others feeling unsafe or unwelcome. 
- We are a diverse group of different backgrounds with differing opinions. 
- We are focused on a unified vision, and the good things we can accomplish together. 
- When giving feedback, _ask for consent first_. "Are you ready to receive a little feedback?" That can make a major difference. 
- We come to this community for various reasons. For some of us, the community is healing and empowering. 
- We want to be excited to work together and to see each other. 
- Please be mindful of how you are interacting with others in the community, and how it effects there.
